mlir/lib/Dialect/Rock/Transforms/FoldTranspose.cpp

```diff
@@ -58,59 +58,41 @@ struct InlineViewLikeOperandsLinalgRewritePattern
 // post-reassociation --> then, produce the set of resultant AffineExprs after
 // the reassociation.
 static void getDelinearizedAffineExpr(
-    ArrayRef<int64_t> originalStridesDimsBeingReassociated,
```

Comment:
Just a note on all the changes. This what I started with but ended up adding what's being removed here to keep the maps invertible AFAIK. However, if you manage to get them inverted -- it is more than welcome.

Reply:
Yes, AffineMap inversion is very limited. So I dropped it and here do explicit low-to-high and high-to-low.